Jenkinsfile

```diff
@@ -270,10 +270,8 @@ try {
             -DUSE_CUDNN=1              \
             -DCMAKE_BUILD_TYPE=Release \
             """
-            def flag = """             \
-            -j\$(nproc)
-            """
-          cmake("build_cuda", defines, flag)
+            def flag = "-v"
```

Comment:
Is the number of build threads automatically determined?

Reply:
@marcoabreu affirmative